src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -356,17 +356,8 @@ class LocationDetails extends Component {
       }
 
       const payload = {
-        name: values.name,
-        active: values.active,
-        locationNumber: values.locationNumber,
-        'organization.id': values.organization.id,
-        'locationGroup.id': _.get(values.locationGroup, 'id') || '',
-        'manager.id': _.get(values.manager, 'id') || '',
-        'locationType.id': _.get(values.locationType, 'id') || '',
+        ...values,
         supportedActivities: _.map(values.supportedActivities, val => val.value),
-        logo: values.logo,
-        bgColor: values.bgColor || '',
-        fgColor: values.fgColor || '',
```

Comment:
have you made sure that it doesn't matter if you pass `''` or `undefined` for those?

Reply:
It doesn't matter. It's just null.
![image](https://github.com/openboxes/openboxes/assets/83239466/41a69fa1-5112-4398-8c1f-5a358d8496d5)

Reply:
I am not sending those properties, so they're not being assigned to the Location object. In the domain we have:
```
    String fgColor = "000000"
    String bgColor = "FFFFFF"
```
So we're not overriding it, because of not sending nulls. And we didn't fail our validation, because we are using those default values.